### ISSUE TITLE
Training on custom data.

### ISSUE THREAD

**Issue by vishwasgpai** [UKPLab/sentence-transformers#63]
was training the model on custom data using this code.

```
data = pd.read_csv("texts.csv")
data = data[['title']].head(1)
embedder = SentenceTransformer('bert-base-nli-mean-tokens')

train_dataloader = DataLoader(data, shuffle=True)
train_loss = losses.CosineSimilarityLoss(model=embedder)

dev_dataloader = DataLoader(data, shuffle=False)
evaluator = EmbeddingSimilarityEvaluator(dev_dataloader)

embedder.fit(train_objectives=[(train_dataloader, train_loss)],
          evaluator=evaluator)


```

But i am getting the this error.

![image](https://user-images.githubusercontent.com/22583613/69426808-1b9e6200-0d54-11ea-8436-89a1f68cacea.png)

**Comment by vishwasgpai** [comment]
The **texts.csv** contains a column of texts that's all.

**Comment by nreimers** [comment]
Please have a look at the respective examples.
For the cosine similarity loss, you must have sentence pairs together with a score (e.g. 0 ... 1) that indicates the similarity of the two sentences.

Fine-tuning with only a set of text is not possible. You always need some pair or triplet structure in order to fine-tune the siamese network structure.

Best 
Nils Reimers
